packages/kbn-profiling-utils/common/frame_group.ts

```diff
@@ -44,5 +61,10 @@ export function createFrameGroupID(
     return `elf;${fileID};${functionName}`;
   }
 
+  const key = `${sourceFilename}:${functionName}`
+  if (equivalenceMap.has(key)) {
+    var replacementString = equivalenceMap.get(`${sourceFilename}:${functionName}`);
```

Comment:
Now you can reuse the key variable here too.
```suggestion
    var replacementString = equivalenceMap.get(key);
```